### DISCUSSION TITLE
Custom field for more than 255 characters

### DISCUSSION THREAD

**Discussion by gianpietro1** [AzuraCast/AzuraCast#6009] [Q&A] [answered]
Hi! I'm using a custom field for storing the artists bio, so it might take up more than 255 characters, which seems to be the length supported today. Do you know if there's any safe way to modify that length?
Thanks!

**Comment by BusterNeece** [comment, marked as answer]
Right now, there isn't a safe way to expand that field size, because it would cause any custom fields to take up significantly larger space in the database, so we went with the trade-off of 255 characters to allow for most use cases.

**Reply by gianpietro1** [reply]
Got it, thanks @BusterNeece !
